module-utils/utility/tests/unittest_utils.cpp

```diff
@@ -526,6 +526,39 @@ TEST_CASE("Byte to hex")
     }
 }
 
+TEST_CASE("Starts with")
+{
+    SECTION("Empty string")
+    {
+        REQUIRE((utils::startsWith("", "abc") == false));
+    }
+
+    SECTION("Empty prefix")
+    {
+        REQUIRE((utils::startsWith("abc", "") == true));
+    }
+
+    SECTION("Both empty")
+    {
+        REQUIRE((utils::startsWith("", "") == true));
+    }
+
+    SECTION("No")
+    {
+        REQUIRE((utils::startsWith("Abcde", "bcd") == false));
+    }
+
+    SECTION("Yes")
+    {
+        REQUIRE((utils::startsWith("Abcde", "Ab") == true));
+    }
```

Comment:
I know it might be sufficient, but I think that it may be more verbose (I mean section names :D )

Reply:
Done.